test/e2e_flags.go

```diff
@@ -68,5 +71,14 @@ func initializeServingFlags() *ServingEnvironmentFlags {
 	flag.BoolVar(&f.DisableLogStream, "disable-logstream", false,
 		"Set this flag to disable streaming logs from system components")
 
+	flag.StringVar(&f.TestNamespace, "test-namespace", "serving-tests",
+		"Set this flag to change the default namespace for running tests.")
+
+	flag.StringVar(&f.AltTestNamespace, "alt-test-namespace", "serving-tests-alt",
+		"Set this flag to change the alternative namespace for running tests.")
+
+	flag.StringVar(&f.TLSTestNamespace, "tls-test-namespace", "tls",
+		"Set this flag to change the namespace for running TLS tests.")
+
```

Comment:
Can we preserve the original defaults:

`serving-tests`
`serving-tests-alt`
`tls`

Reply:
They're preserved. Or no? The default values are `serving-tests`, `serving-tests-alt`, `tls` . Am I missing something?

Reply:
sorry I misread the flag name as default value